.github/workflows/test.yml

```
@@ -38,7 +38,7 @@ jobs:
                   git config --global core.autocrlf false
                   git config --global core.eol lf
             - name: Install Go
-              uses: actions/setup-go@v2-beta
+              uses: actions/setup-go@v2.1.3
```

just to get security updates
```suggestion
              uses: actions/setup-go@v2
```